diagnostics/model_test/verify_model.py

```diff
@@ -223,20 +225,29 @@ def parse_json():
         l2_norm = calculate_norm(result_ngraph, result_tf_graph, 2)
         inf_norm = calculate_norm(result_ngraph, result_tf_graph, np.inf)
 
```

Comment:
Yes, its a good idea to group by tensorname and then display the norms.

The printing of the 3 norms have a lot of repeat code, might I suggest:
```
th_dict = {"L1": l1_norm_threshold, "L2": l2_norm_threshold, "Linf": inf_norm_threshold}

#start the loop and compute norms
norm_dict = {"L1": l1_norm, "L2": l2_norm, "Linf": inf_norm}
print("\n["+tname+"]")
for norm_name in norm_dict:
    if norm_dict[norm_name] > th_dict[norm_name]:
        print("The ",norm_name," norm %f is greater than ",norm_name," threshold %f" % (norm_dict[norm_name], th_dict[norm_name]))
        passed = False
    else:
        #the other prints here
```

Reply:
Actually there's a request to show the norms even for passed case. So I put it intentionally. Because we want to see how much gap we have compare to the threshold and if needed to adjust the value.

Reply:
I see. Sounds good, then the current format is good.